### PR TITLE
enable generation of adapter and template .proto files

### PR DESCRIPTION
**Pull request by theganyo**
Replaces https://github.com/istio/vendor-istio/pull/6

### CODE REVIEW

**Comment by ldemailly** [comment]
thanks, which pr will use this ? can you link it ?

and the output of
`make vendor.check depend.status`

File: Gopkg.lock [hunk 1]
```diff
@@ -126,8 +126,8 @@
     "private/protocol/xml/xmlutil",
     "service/sts"
   ]
-  revision = "f7f1ee9550cc76ceea731d99c8dbe00a0c39f66d"
-  version = "v1.13.4"
+  revision = "adeb60566bc8c9202b0f1be7e3a675beedbc11f0"
+  version = "v1.13.2"
```
Comment (ldemailly) [inline Gopkg.lock:130]: it’s a bit odd there is a downgrade here no?

File: Gopkg.lock [hunk 2]
```diff
@@ -292,11 +292,10 @@
...
     "internal/pool",
     "internal/proto",
-    "internal/singleflight",
-    "internal/util"
+    "internal/singleflight"
   ]
-  revision = "56dea1f39a5f4a64fe8c639464c966c75f226e3e"
-  version = "v6.9.1"
+  revision = "fa7f64f7f27348658ecfa71dd71dfb2d112e2f86"
+  version = "v6.9.0"
```
Comment (ldemailly) [inline Gopkg.lock:298]: ditto

**Comment by theganyo** [comment]
This was for PR https://github.com/istio/istio/pull/3744.

File: Gopkg.toml [hunk 3]
```diff
@@ -105,13 +106,17 @@ required = [
 #  name = "k8s.io/ingress-nginx"
 #  revision = "0c6f15e372c831de52fcc393932540bb3a6d51b5"
 
+[[override]]
+  name = "github.com/ugorji/go"
+  revision = "708a42d246822952f38190a8d8c4e6b16a0e600c"
```
Comment (ldemailly) [inline Gopkg.toml:111]: you seem based on an old toml

please start from the current one in istio/istio master

**Review by ldemailly — CHANGES_REQUESTED** [review]

**Comment by theganyo** [comment]
I just reran the generate. I thought that's what you wanted. But apparently everything needs to start over. Will do that.

**Comment by ldemailly** [comment]
sorry I wasn’t clear on how you got a conflict but it seems it is simply because your istio/istio branch is outdated, you need to update it before you rerun

**Comment by theganyo** [comment]
Ok, I can't get the `make depend.update` to run now. I'm just going to close this and add my changes back in on a new branch.

**Pull request closed without merging**